src/actions/vstudio/_vstudio.lua

```diff
@@ -338,7 +338,7 @@
 		end
 
 		-- Then the system libraries, which come undecorated
-		local system = config.getlinks(cfg, "system", "fullpath")
+		local system = config.getlinks(cfg, "system", "name")
```

Comment:
yeah, we generate a lot of this stuff through the package system, maybe I should have a look at how that is happening right now...